### DISCUSSION TITLE
Force user to login again if Refresh token is invalid/expires

### DISCUSSION THREAD

**Discussion by hiimnhan** [Flyrell/axios-auth-refresh#245] [Q&A]
How can I handle force user to login again if Refresh token is invalid/expires. I see the function is just getting new access token but where can I handle case that Refresh token is invalid or expired?

**Comment by bryanprimus** [comment]
have you found a workaround to this?

**Comment by bryanprimus** [comment]
you can probably redirect user in the catch block of `refreshAuthLogic` like mentioned by the author here https://github.com/Flyrell/axios-auth-refresh/issues/128#issuecomment-856581472
